gdpr/README.MD

```diff
@@ -0,0 +1,79 @@
+# Kukkuu GDPR API demo
```

Comment:
Now that you created a separate readme for the GDPR related stuff, maybe could you please move the GDPR API data export graph from the main readme to the gdpr readme as it seems like the better place for that info?